src/OpenTelemetry.Exporter.Geneva/OpenTelemetry.Exporter.Geneva.csproj

```diff
@@ -16,6 +16,7 @@
 
   <ItemGroup>
     <PackageReference Include="OpenTelemetry" Version="$(OTelSdkVersion)" />
+    <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.0.0" Condition="'$(TargetFramework)' == 'net462'" />
```

Comment:
With `1.7.0-rc.1` version of the SDK, this package now pulls in `8.0.0` version of `Microsoft.Extensions.Logging.Configuration` instead of `3.1.0` version that used to come with the previous dependency.

`Microsoft.Extensions.Logging.Configuration` has added a dedicated net462 target in `8.0.0` which does not have the `RuntimeInformation` library included. That's why we need an explicit package dependency for `net462` targets now. With `3.1.0` version, there was no dedicated `net462` target, and the package ended up using `netstandard2.0` bits which includes the `RuntimeInformation` package.

Follow this discussion for more details: https://github.com/open-telemetry/opentelemetry-dotnet/pull/5020#discussion_r1380902962